src/directives/async-append.ts

```diff
@@ -84,7 +84,7 @@ export const asyncAppend = directive(
         // Check to see if we have a previous item and Part
         if (itemPart !== undefined) {
           // Create a new node to separate the previous and next Parts
-          itemStartNode = createMarker();
+          itemStartNode = createStartMarker();
```

Comment:
Don't we need to insert end markers, too?

Reply:
True. This change was just to get the branch building; we should actually update this in a separate PR, or on the `hydration` branch.